<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowLevelQWidget</class>
 <widget class="QWidget" name="WindowLevelQWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Window - Level</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4314</number>
   </property>
   <property name="sliderPosition">
    <number>400</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>150</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Window:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLevel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Level:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderLevel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="minimum">
    <number>-3024</number>
   </property>
   <property name="maximum">
    <number>1290</number>
   </property>
   <property name="value">
    <number>40</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>150</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelPreset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Preset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPresets">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brain</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Abdomen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mediastinum</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lung</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MIP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelWindowValue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLevelValue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderLevel</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>labelLevelValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderLevel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelLevelValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderWindow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelWindowValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderWindow</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>labelWindowValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>

